Sources/App/Views/Utils.swift

```diff
@@ -10,3 +10,18 @@ func pluralize(count: Int, singular: String, plural: String? = nil) -> String {
             return "\(count) \(plural)"
     }
 }
+
+
+extension String {
+    func pluralized(for count: Int, plural: String? = nil) -> String {
+        let plural = plural ?? self + "s"
+        switch count {
+            case 0:
```

Comment:
It's only when you see lines of code like this that you're reminded how ridiculous English is 😂

Reply:
Let me tell you about German. I'm glad we're not localising this 🤣🤣